DNN Platform/DotNetNuke.Abstractions/Logging/ILogProperties.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System;
+    using System.Collections;
+    using System.Xml;
+
+    /// <summary>
+    /// add xml docs.
+    /// </summary>
+    public interface ILogProperties : IList, ICollection, IEnumerable, ICloneable
```

Comment:
Could we implement `ICollection<ILogDetailInfo>`?

Reply:
This makes sense, I didn't create that interface and it should be added